### PR TITLE
Feature/aquality tracking

### PR DESCRIPTION
**Pull request by paveliam**

### CODE REVIEW

**Review by knysh — APPROVED** [review]

File: Aquality.Selenium.Template/Aquality.Selenium.Template.SpecFlow/specflow.json [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
   "stepAssemblies": [
     { "assembly": "Aquality.Selenium.Template.SpecFlow" },
-    { "assembly": "Allure.SpecFlowPlugin" }
+    { "assembly": "Allure.SpecFlowPlugin" },
+    { "assembly": "AqualityTracking.SpecFlowPlugin" }
   ],
   "runtime": {
     "stopAtFirstError": true,
     "missingOrPendingStepsOutcome": "Error"
   }
```
Comment (mialeska) [inline Aquality.Selenium.Template/Aquality.Selenium.Template.SpecFlow/specflow.json:10]: Don't we need section "plugins"
`
  "plugins": [
    {
      "name": "Allure",
      "type": "Runtime"
    }
  ],`
here?

File: Aquality.Selenium.Template/Aquality.Selenium.Template.SpecFlow/Aquality.Selenium.Template.SpecFlow.csproj [hunk 2]
```diff
@@ -28,6 +29,9 @@
     <None Update="allureConfig.json">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
+    <None Update="Resources\aqualityTracking.json">
```
Comment (mialeska) [inline Aquality.Selenium.Template/Aquality.Selenium.Template.SpecFlow/Aquality.Selenium.Template.SpecFlow.csproj:32]: Do we have ability to copy the config template after adding a nuget package like Allure does?

**Review by mialeska — APPROVED** [review]

File: Aquality.Selenium.Template/Aquality.Selenium.Template.SpecFlow/Aquality.Selenium.Template.SpecFlow.csproj (hunk 2 above)
Comment (paveliam) [reply Aquality.Selenium.Template/Aquality.Selenium.Template.SpecFlow/Aquality.Selenium.Template.SpecFlow.csproj:32]: Actually, it does, But we cannot use copied file. We have to create our own. That's why I've added `aqualityTracking.Template.json` to AqualityTracking,SpecFlowPlugin package. If you open solution in VS you'll see this file copied from nuget automatically.

File: Aquality.Selenium.Template/Aquality.Selenium.Template.SpecFlow/specflow.json (hunk 1 above)
Comment (paveliam) [reply Aquality.Selenium.Template/Aquality.Selenium.Template.SpecFlow/specflow.json:10]: No, it is not necessary, only assembly.

**Pull request merged**